nautobot/core/tests/test_api.py

```diff
@@ -72,6 +76,9 @@ def setUp(self):
         self.cf_text.save()
 
     def test_api_docs(self):
+        user = User.objects.create_user(username="nautobotuser")
+        self.client.force_login(user)
```

Comment:
```suggestion
        self.client.force_login(self.user)
```

Reply:
`APIDocsTestCase(TestCase)` inherits from Django TestCase, which do not have `self.user`